webcdi/cdi_forms/views.py

```diff
@@ -398,9 +398,13 @@ def cdi_form(request, hash_id):
                         if not payment_code.objects.filter(hash_id = hash_id).exists():
                             if administration_instance.study.name == "Wordful Study (official)": # for wordful study: if its second admin, give 25 bucks else 5
                                 if administration_instance.repeat_num == 2:
-                                    gift_amount_search = 25
+                                    # if this subject already has claimed $25: give them $5 this time
+                                    if payment_code.objects.filter(hash_id=administration_instance.url_hash, gift_amount=25.0):
```

Comment:
what is the return type of the filter statement? / is it appropriate for the Boolean evaluation?